sync/Cargo.toml

```diff
@@ -11,5 +11,11 @@ description = "Synchronization primitives for SGX enclaves"
 categories = ["hardware-support", "no-std"]
 keywords = ["sgx", "no-std"]
 
+[lib]
+# test false due to needing an enclave to fully link
+test = false
+doctest = false
+
 [dependencies]
 mc-sgx-panic = { path = "../panic", version = "=0.1.0" }
+mc-sgx-tstdc = { git = "https://github.com/mobilecoinfoundation/sgx", rev = "12c88f98c6c5b7b71ee7cfe3579422e08eb52cf5" }
```

Comment:
This should get changed prior to integration.  It depends on https://github.com/mobilecoinfoundation/sgx/pull/256

Reply:
This has been fixed now